### ISSUE TITLE
SpringReverb: call of overloaded 'min(size_t&, size_t)' is ambiguous

### ISSUE THREAD

**Issue by phdsg** [VCVRack/Befaco#9]
maybe related to https://github.com/VCVRack/Rack/commit/5bc9a696b090d223ad5c28c7aa4cf59db7410be5

```
g++ -Wsuggest-override -std=c++11 -I./pffft -DPFFFT_SIMD_DISABLE -fPIC -I../../include -I../../dep/include -DSLUG=Befaco -DVERSION=0.6.0 -MMD -MP -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_WIN -D_USE_MATH_DEFINES -c -o build/src/SpringReverb.cpp.o src/SpringReverb.cpp
src/SpringReverb.cpp: In member function 'void RealTimeConvolver::setKernel(const float*, size_t)':
src/SpringReverb.cpp:89:52: error: call of overloaded 'min(size_t&, size_t)' is ambiguous
    size_t len = min(blockSize, length - i*blockSize);
                                                    ^
In file included from ../../include/util/common.hpp:52:0,
                 from ../../include/rack.hpp:4,
                 from src/Befaco.hpp:1,
                 from src/SpringReverb.cpp:2:
../../include/util/math.hpp:21:12: note: candidate: int rack::min(int, int)
 inline int min(int a, int b) {
            ^~~
../../include/util/math.hpp:61:14: note: candidate: float rack::min(float, float)
 inline float min(float a, float b) {
              ^~~
make: *** [../../compile.mk:58: build/src/SpringReverb.cpp.o] Error 1
```

**Comment by MarcBoule** [comment]
Getting the same error here also. Win10, dev build.

**Comment by AndrewBelt** [comment]
I've made a commit 6f36a77 to Rack which makes min, max, and clamp generic, but the types of the arguments all need to match correctly. Try compiling some plugins and see if this change is a complete disaster, and I'll revert in a few hours when I'm back from an event. This keeps Befaco broken and breaks Audible Instruments, but I'm concerned about third-party plugins.

**Comment by MarcBoule** [comment]
Fixed in Befaco, but Fundamental now seems to have a perhaps related problem:
https://github.com/VCVRack/Rack/issues/810
And similar with Audible.

**Comment by cschol** [comment]
@AndrewBelt Yes. This is the same problem. I counted 18/37 build failures in the `v2` repo.

**Comment by AndrewBelt** [comment]
Fixed in e74ad7f.
